content/sensu-core/2.0/reference/entities.md

```diff
@@ -38,7 +38,7 @@ sensu-agent start --id hostname --environment default --organization default --k
 
 ### What is a proxy entity?
 
-A proxy entity is a Sensu entity which falls under the "proxy" class. Proxy entities (formerly known as proxy clients, "Just-in-time" or "JIT" clients) are dynamically created entities, added to the entity registry if an entity does not already exist for a check result. Proxy entity registration differs from keepalive-based registration because the registration event happens while processing a check result (not a keepalive message). Sensu proxy entities allow Sensu to monitor external resources (e.g. on systems and/or devices where a sensu-agent cannot be installed, such a network switches), using the defined check ProxyEntityID to create a proxy entity for the external resource. Once created, proxy entities work much in the same way as any other Sensu entity.
+Proxy entities (formerly known as proxy clients, "Just-in-time" or "JIT" clients) are dynamically created entities, added to the entity registry if an entity does not already exist for a check result. Proxy entity registration differs from keepalive-based registration because the registration event happens while processing a check result (not a keepalive message). Sensu proxy entities allow Sensu to monitor external resources (e.g. on systems and/or devices where a sensu-agent cannot be installed, such a network switches), using the defined check ProxyEntityID to create a proxy entity for the external resource. Once created, proxy entities work much in the same way as any other Sensu entity.
```

Comment:
First time I'm hearing of the registry. Is that something that could be intro'ed in "What is a Sensu entity?" Maybe something like: "Entities are stored in a registry managed by xyz" 

Could be the last sentence of that entry.